point2D_intList polygonPoints
int32 type
int32 id
int32[] connectedPolygons
point2D_intList[] connectedPaths
